proto/regen/ecocredit/v1/tx.proto

```diff
@@ -23,6 +23,16 @@ service Msg {
   // be distributed to recipients in either tradable or retired form.
   rpc CreateBatch(MsgCreateBatch) returns (MsgCreateBatchResponse);
 
+  // MintBatch issues new token in a given batch.
+  // Currently the issuer must be the Credit Class issuer, or an address
```

Comment:
The issuer should be the issuer of the original batch. Not any credit class issuer

Reply:
done